tutoroars/templates/oars/apps/vector/local.toml

```diff
@@ -0,0 +1,16 @@
+{% include "oars/apps/vector/partials/common-pre.toml" %}
+
+### Sources
+# Capture logs from all docker containers
+[sources.docker_logs]
+type = "docker_logs"
+[transforms.openedx_containers]
+type = "filter"
+inputs = ["docker_logs"]
+condition = 'includes(["lms", "cms", "lms-worker"], .label."com.docker.compose.service")'
+[transforms.lms_worker]
```

Comment:
It's likely this isn't needed and we can just use `openedx_containers`, but during testing the xapi regex failures were so numerous it made developement impossible.

Reply:
Remove this and am just using `openedx_containers` now.